IdentityCore/src/MSIDOAuth2Constants.m

```diff
@@ -125,6 +126,7 @@
 NSString *const MSID_LAST_MOD_APP_CACHE_KEY              = @"last_modification_app";
 NSString *const MSID_APPLICATION_IDENTIFIER_CACHE_KEY    = @"application_cache_identifier";
 NSString *const MSID_ACCESS_TOKEN_CACHE_TYPE             = @"AccessToken";
+NSString *const MSID_ACCESS_TOKEN_WITH_AUTH_SCHEME_CACHE_TYPE             = @"accesstoken_with_authscheme";
```

Comment:
Design references to this type as "AccessToken_With_AuthScheme". Can you please double check exact string?

Reply:
Good catch. Addressed.